vic/drivers/image/Makefile

```diff
@@ -75,6 +75,7 @@ INCLUDES = -I ${DRIVERPATH}/include \
 
 # Uncomment to include debugging information
 CFLAGS  =  ${INCLUDES} ${NC_CFLAGS}  -ggdb -O0 -Wall -Wextra -std=c99 \
+					 -fopenmp \
```

Comment:
right - and as @jhamman and I just discussed, we need to include the flag even if we do not use the `openmp` features, but simply rely on `mpi` for parallelization.